ResearchKit/Common/ORKFormItemCell.m

```diff
@@ -1148,3 +1156,125 @@ - (BOOL)textField:(UITextField *)textField shouldChangeCharactersInRange:(NSRang
 }
 
 @end
+
+
+#pragma mark - ORKFormItemLocationCell
+
+@interface ORKFormItemLocationCell () <ORKLocationSelectionViewDelegate>
+
+@property (nonatomic, assign) BOOL editingHighlight;
+
+@end
+
+
+@implementation ORKFormItemLocationCell {
+    ORKLocationSelectionView *_selectionView;
+    NSLayoutConstraint *_heightConstraint;
+}
+
+- (void)cellInit {
+    [super cellInit];
+    
+    _selectionView = [[ORKLocationSelectionView alloc] initWithOpenMap:NO
+                                                    useCurrentLocation:((ORKLocationAnswerFormat *)self.formItem.answerFormat).useCurrentLocation
+                                                edgeToEdgePresentation:NO];
+    _selectionView.delegate = self;
+    
+    [self.contentView addSubview:_selectionView];
+
+    if (self.formItem.placeholder != nil) {
+        [_selectionView setPlaceholderText:self.formItem.placeholder];
+    }
+    
+    [self answerDidChange];
```

Comment:
Normally you don't need to call `answerDidChange` at here, it has been handled by general initializer.